system_baseline/openapi/internal_api.spec.yaml

```diff
@@ -44,6 +44,51 @@ paths:
           $ref: '#/components/responses/Forbidden'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /systems/{system_ids}:
+    parameters:
+      - $ref: '#/components/parameters/SystemIds'
+    delete:
+      summary: delete one or more systems
+      description: "delete one or more systems"
+      operationId: system_baseline.internal_views.v1.delete_systems_by_ids
+      responses:
+        '200':
+          description: a success message
+          content:
+            application/json:
+              schema:
+                type: string
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+  /systems/deletion_request:
```

Comment:
do we need both the POST delection_request as well as the delete on the systems/{system_ids} endpoint above?

Reply:
explained above